cmd/ui/src/views/Explore/GraphView.tsx

```diff
@@ -139,6 +141,8 @@ const GraphView: FC = () => {
     const opts: GlobalOptionsState = useSelector((state: AppState) => state.global.options);
     const formIsDirty = Object.keys(useSelector((state: AppState) => state.tierzero).changelog).length > 0;
     const [graphologyGraph, setGraphologyGraph] = useState<MultiDirectedGraph<Attributes, Attributes, Attributes>>();
+    const [currentNodes, setCurrentNodes] = useState<GraphNodes>({});
```

Comment:
That is the case, but to get it in the format we want and filter out non-nodes, we'd have to call `transformFlatGraphResponse` on it again before passing it in (or something like it). Since we were already doing that in the first `useEffect` I just grabbed the nodes from there, but if you think its better to handle that logic separately I can do that too.

Reply:
I see what you mean. This does make it so we don't have to filter that out again. Thanks for pointing that out.

Reply:
transformFlatGraphResponse makes me sad. Longer term we should consider a refactor to homogenize how we store graph information in the global store.